docs/manuals/ibridges.qmd

```diff
@@ -2,24 +2,84 @@
 title: Using iBridges on SURF Research Cloud
 ---
 
-The instructions below describe the process of data transfer between SURF Research Cloud and Yoda using iBridges. iBridges is a command-line tool for transfering data to and from Yoda (and other iRODS servers), focused on user-friendliness. It is developed and supported by the Utrecht University Research Engineering team. Like the [iCommands suite](icommands.qmd), it is suitable for transferring large amounts of data, but it is easier to use.
+The instructions below describe the how to setup iBridges to transfer data between SURF Research Cloud and Yoda or any other iRODS instance. iBridges consists of three parts:
+- a python package 
+- a command line interface (CLI)
+- a graphical user interface (GUI)
 
-Note: a graphical desktop version of iBridges is currently under development, but not yet supported on ResearchCloud.
+With all three parts you can safely transfer your data between Yoda and your workspace.
+
+It is developed and supported by the Utrecht University Research Engineering team. Like the [iCommands suite](icommands.qmd), it is suitable for transferring large amounts of data, but it is easier to use.
+
+The python package and the command line interface are available or can be made available on all workspaces while the graphical user interface will only work on desktop workspaces, i.e. workspaces where you will have a Windows or linux Desktop. 
 
 ### Prerequisites
 
-You will need some basic Linux (commandline) skills to be able to use iBridges.
+You will need access to a Yoda or other iRODS instance which hosts your data.
 
 iBridges is already installed on many SURF ResearchCloud workspaces developed by Utrecht University. You can check if it is present by simply typing the following command on your terminal: `ibridges`. If iBridges is installed, this should show you the program's help menu.
 
-If you need to install iBridges, you can execute: `pip install ibridges`.
+- If you need to install iBridges, you can execute: `pip install ibridges`.
+- On desktop workspaces you can execute: `pip install ibridgesgui`.
 
 ### Creating an iRODS environment file
```

Comment:
I suggest having separate pages for instructions for the gui and for python/CLI. Should we create two separate manuals? Or have one with the sections above here, and then have two links here to ibridges GUI and ibridges CLI/python?